### PR TITLE
UX/UI corrections pt.1

### PR DESCRIPTION
**Pull request by Jorge-N**
- [fix merge mistake (made by our junior dev Valen Schiaffino)](https://github.com/vSchiaffino/apps-moviles/commit/d7f36f823d754335c90498fedd3bf6e3b5eb54f4)
- [borderRadius on SelectItems](https://github.com/vSchiaffino/apps-moviles/commit/310d21dc3be30d45bf5c2c7a9c742bd733eaacc8)
- [add animated label on OutlineSelect](https://github.com/vSchiaffino/apps-moviles/commit/0dc7fcc2056b38b761aa373f23e7e07afebc1b15)
- [add animated label on OutlinedInput](https://github.com/vSchiaffino/apps-moviles/commit/23407c29c69262a8e5fb2f277309ac2e9788a17f)
- [add back borderRadius to ProfilePic Modal](https://github.com/vSchiaffino/apps-moviles/commit/6cd593dc55a017e4f1e0c98374b5d20382fc6285)
- [warehouse-transfer from label change](https://github.com/vSchiaffino/apps-moviles/commit/2cfd97a039f38250d34030cdbe5d9790fbed9b17)
- [revert testing commits](https://github.com/vSchiaffino/apps-moviles/pull/7/commits/0657b24e112a115279920d00aad3badf8334e8f3)

### CODE REVIEW

File: app/index.tsx [hunk 1]
```diff
@@ -3,14 +3,15 @@ import Container from '@/components/Container'
...
 import { Redirect, router } from 'expo-router'
 import React from 'react'
 import { View } from 'react-native'
 
-const Dashboard = () => {
-  const { user } = useUser()
-  if (!user) return <Redirect href="/login" />
+const Dashboard = (user: UserPayload) => {
+  // const { user } = useUser()
+  // if (!user) return <Redirect href="/login" />
```
Comment (vSchiaffino) [inline app/index.tsx]: Descomenta esto

File: app/profile.tsx [hunk 2]
```diff
@@ -4,9 +4,20 @@ import { Redirect } from 'expo-router'
...
+    <ProfilePage
+      user={{
+        id: 1,
+        name: 'Valentin',
+        lastName: 'Schiaffino',
+        mail: 'valenschg@gmail.com',
+        user: 'valen',
+      }}
+    />
+  )
```
Comment (vSchiaffino) [inline app/profile.tsx]: Junior

File: components/OutlinedSelect/OutlinedSelect.tsx [hunk 3]
```diff
@@ -30,18 +30,33 @@ const OutlinedSelect: React.FC<OutlinedInputProps> = ({
...
+      duration: 50,
+      useNativeDriver: false,
+    }).start()
+  }
+
+  const labelPosition = animatedValue.interpolate({
+    inputRange: [0, 1],
+    outputRange: [19, -7],
+  })
+
```
Comment (vSchiaffino) [inline components/OutlinedSelect/OutlinedSelect.tsx:53]: Esto lo podes extraer en un hook, es igual que el del OutlinedInput

**Review by vSchiaffino — DISMISSED** [review]

File: app/index.tsx (hunk 1 above)
Comment (Jorge-N) [reply app/index.tsx]: no
👍

**Comment by Jorge-N** [comment]
dale acepta @vSchiaffino

**Review by vSchiaffino — APPROVED** [review]
Sos el mejor desarrollador de la historia

**Pull request merged**